예산.js

```diff
@@ -0,0 +1,14 @@
+function solution (d, budget) {
+    let answer = 0, sum = 0;
+    d.sort ((a,b) => a - b);
```

Comment:
for, if같은 조건문은 한 칸 띄우고 조건식을 걸지만
sort, filter, map같은 메서드는 공백없이 들어가요~
```suggestion
    d.sort((a,b) => a - b);
```